content/configuration/policy_in_gocd.md

```diff
@@ -0,0 +1,67 @@
+---
+description: GoCD authorization via policy
+keywords: GoCD configuration, authorization, administration, users, roles, role-based security, security, security configuration, user permissions, GoCD administration, role-based access control, rbac
+title: Policy in GoCD
+---
+
+# Policy
+
+A policy is a set of permissions for GoCD entities. This helps restricts the entity access based on the user's role within GoCD. GoCD supports a restrictive model of access i.e. until access is granted, the entity can't be accessed by anyone (except admins).
+
+A permission consists of 3 values:
+
+ 1. __Type__: The type of GoCD entity. Supported values are `environment`
```

Comment:
We also have (*). We should mention that it would mean all supported entities, for now, only Environments is supported.